contrib/contrib_build_config.bzl

```diff
@@ -0,0 +1,8 @@
+# See bazel/README.md for details on how this system works.
+CONTRIB_EXTENSIONS = {
+    #
+    # HTTP filters
+    #
+
+    "envoy.filters.http.squash":                        "//contrib/filters/http/squash/source:config",
```

Comment:
Just so that I understand this right: `bazel --//contrib/filters/http/squash:enabled ...` would enable this contrib extension from the bazel command line when building something else than `//contrib/exe/envoy-static` build target?

Reply:
I'm not sure. I need to test this. I'm pretty sure it won't work outside of this target, but it will work when building the contrib exe to customize that build. I will try it out and report back.

Reply:
`bazel build //contrib/exe:envoy-static --//contrib/filters/http/squash/source:enabled=false` works properly, but trying to enable the contrib filter this way on top of `//source/exe:envoy-static` does not work. I will make this clear in the docs.